src/main/java/use_case/suggested_locations/SuggestedLocationInteractor.java

```diff
@@ -0,0 +1,21 @@
+package use_case.suggested_locations;
+
+/**
+ * The Suggested Locations Interactor.
+ */
+public class SuggestedLocationInteractor implements SuggestedLocationInputBoundary {
+
+    private final SuggestedLocationOutputBoundary outputBoundary;
+
+    public SuggestedLocationInteractor(SuggestedLocationOutputBoundary outputBoundary) {
+        this.outputBoundary = outputBoundary;
```

Comment:
this should be setting a presenter = output boundary since output boundary is an interface and presenter is the implementation. see the suggestlocationinteractor for example